### DISCUSSION TITLE
Welcome to Litematica Discussions!

### DISCUSSION THREAD

**Discussion by maruohon** [maruohon/litematica#267] [General]
## 👋 Welcome to Litematica discussions!
  - Here you can ask questions or give feedback without having to open actual new issues for everything.
  - Another place for more real-time discussions is on my Discord server: https://discord.gg/2FgywHj

**Comment by Orca133** [comment]
My schematic wont like show up in my one specific world

**Reply by dosetapped** [reply]
try to unload it and reload it?

**Comment by Josh6589** [comment]
whew do i save my schematic to 
i dont have a folder to save it to on my minecraft folder on configs

**Reply by Samalando** [reply]
Then make one!

**Reply by maruohon** [reply]
Your launcher may be using a different location for the game directory than the vanilla launcher location.

You can get to the game directory by opening the Resource Packs menu in-game, and then click on Open Folder, and then go one directory up. The `schematics` directory should be there then.

**Comment by nahar64** [comment]
i am not sure this is the right place, but anyway:
i am building a large build (map art), and almost all of it is completed, but there are some blocks that are missing and i don't know were they supposed to go. is there a way to highlight the blocks i didn't placed yet, or i need to go all over it block by block myself to see what is missing?

**Reply by maruohon** [reply]
You can use the Schematic Verifier to check for any mistakes. You can open it via the placement configuration menu, or I think the default hotkey is `M + V`. When you start it and it has finished, you can select things from the list to highlight, either individual block mismatch pairs, or the "category headers" such as "Wrong State" or "Missing Block".

Note that you have to have that placement selected for the in-world markers to render. For example select the placement via the Schematic Placements menu, so that it has the white selection outline.

**Comment by AEH7131** [comment]
Does anyone know why the containers are being filled in when i paste a schematic into my world i am on 1.26.2 and it is a single player creative world

**Reply by maruohon** [reply]
Are or aren't?

All NBT data that the schematic has is supposed to get restored when pasting in single player. This includes any inventory contents.

But if a schematic is saved from a server without "the support bits in place" (in 1.21+ Servux on server and Generic -> `entityDataSync` is enabled, or no Servux but player is OP'd and `entityDataSyncFallback` is enabled) then by default it won't include any inventory contents, as the vanilla server does not send those to the client, except when you open an inventory screen (and then it forgets them immediately when you close it).

**Reply by AEH7131** [reply]
aren't (sorry typo)

**Reply by maruohon** [reply]
Does the schematic have the contents saved? Can you see them if you use the Block Info Overlay (by default hold `i` when looking at the schematic blocks).

If it does have them, then which paste type are you using? You haven't enabled the option Generic -> `pasteUsingCommandsInSp` for some reason? Or some of the `pasteIgnore*` options?

**Reply by AEH7131** [reply]
thanks, i had pasteignoreinventories on for some reason
